### PR TITLE
Upgrade to 1.6

### PR DESCRIPTION
**Pull request by vdayanand**
- handle TOML.print behavior in 1.6
- Use Downloads.download instead of HTTP.jl, Base.download

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ os:
 services:
   - xvfb
 julia:
-  - 1.4
+  - 1.6
```
Comment (pfitzseb) [inline .travis.yml:7]: Do we want to test on 1.5 here as well or do you think that's gonna take too long?

Comment (pfitzseb) [reply .travis.yml:7]: Actually, we should probably move to GH actions anyways at some point.

**Comment by pfitzseb** [comment]
Can you rebase this on master? Just merged a PR that disables doctests, which are, I think, responsible for the test failures here.

File: .travis.yml (hunk 1 above)
Comment (vdayanand) [reply .travis.yml:7]: Yes, increased CI time is a problem. I don't  mind adding it back though

**Comment by vdayanand** [comment]
Tests passed after updating packages to their latest version.

**Pull request merged**